src/cljs/rems/application.cljs

```diff
@@ -680,12 +680,25 @@
                   [application-link (:application/copied-to event) (text :t.applications/application)]
 
                   (not-empty (:application/comment event)))
-        time (localize-time (:event/time event))]
+        time (localize-time (:event/time event))
+        event-public (:application/public event)]
     [:div.row.event
      [:label.col-sm-2.col-form-label time]
      [:div.col-sm-10
       [:div.event-description.d-flex.justify-content-between.col-form-label
-       [:b.col.pl-0 event-text]]
+       [:div.col.pl-0
+        (cond
+          (and see-everything
+               (true? event-public)) [:<>
+                                      [:i.event-public.fas.fa-eye
+                                       [:span.sr-only (text :t.applications.event/shown-to-applicant)]]
+                                      [:b.pl-2 (text-format :t.label/parens event-text (text :t.applications.event/shown-to-applicant))]]
```

Comment:
Maybe we don't need the text "shown to applicant" as the icon is meant to signifythat. It could be its title instead. Screenreader span is perhaps not required then?